docs/_static/spec/rasa.yml

```diff
@@ -256,7 +256,7 @@ paths:
       - Tracker
       summary: Run an action in a conversation
       description: >-
-        Runs the action, calling the action server if necessary.
+        DEPRECATED. Runs the action, calling the action server if necessary.
```

Comment:
I would also add `deprecated: true` to the operation object itself, OpenAPI allows us to do that (see https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#operation-object)